pkg/fw/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN curl -fsSL ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM434
     curl -fsSL ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM43430B0.hcd -O &&\
     curl -fsSL ${RPI_BT_FIRMWARE_URL}/${RPI_BT_FIRMWARE_VERSION}/broadcom/BCM4345C5.hcd -O
 
-FROM busybox as compactor
+FROM busybox@sha256:ef320ff10026a50cf5f0213d35537ce0041ac1d96e9b7800bafd8bc9eff6c693 as compactor
```

Comment:
Probably it would be better to use `FROM ${EVE_BUILDER_IMAGE} as compactor` to not use additional external entities here.

Reply:
I am fine with that; I was trying not to change anything.

Now that I look at it, it doesn't do anything in `compactor`. It just copies over the various `/lib/firmware/` from the `build` image, and then runs `cp` to create a symlink. Is that it? The entire purpose is for the symlink? If so, then, yes, let's adopt your approach. Doing it now.

Reply:
Done.